src/JuliusSweetland.OptiKey/Services/AutoComplete/BasicAutoComplete.cs

```diff
@@ -50,24 +50,21 @@ public IEnumerable<string> GetSuggestions(string root)
         }
 
         public void AddEntry(string entry, DictionaryEntry newEntryWithUsageCount)
-        {
-
-            //Also add to entries for auto complete
-            var autoCompleteHash = entry.Normalise(log: false);
+		{
+			if (!string.IsNullOrWhiteSpace(entry) && entry.Contains(" "))
```

Comment:
Thought this would make more sense if we keep the logic order the same between  `BasicAutoComplete` and `NGramAutoComplete`, where the phrase was added into the dictionary first. I will take a look at the multi-select issue. Will PR again once that's fixed.